exercises/spiral-matrix/test/Tests.hs

```diff
@@ -0,0 +1,40 @@
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
+
+import Spiral (spiral)
+
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "spiral" $ do
+
+  it "empty spiral" $
+    spiral 0 `shouldBe` []
+
+  it "trivial spiral" $
+    spiral 1 `shouldBe` [ [1] ]
+
+  it "spiral of side 2" $
```

Comment:
Oh, I guess you might want to change it in problem-specifications too, then.

If just saying its size, maybe something like `spiral of side length 2`?

I was going to try to give some thought to "what shall the larger sizes test that smaller sizes don't" and I might have thought of some answer like "size 3 has a middle-of-edge cell", "size 4 has multiple middle-of-edge cells", "size 5 is the smallest matrix that turns back to the starting direction a second time" for those solutions that are fundamentally going from number -> index rather than index -> number but I don't think that distinction is very meaningful for an index -> number solution and might even lead students away from an index -> number solution.

So I have no suggestion other than that.